packages/next-common/utils/constants.js

```diff
@@ -87,8 +87,7 @@ export const BalanceDecimals = {
 };
 
 export const CACHE_KEY = {
-  lastLoginAddress: "lastLoggedInAddress",
-  lastLoginExtension: "lastLoginExtension",
```

Comment:
Why remove this 2 cache keys? Are they used for remembering last login wallet and address, so the login popup will auto select the address? If so, why not keep them?